lib/config/index.js

```diff
@@ -12,4 +30,7 @@ nconf.argv()
     });
 
 nconf.defaults(defaults);
-module.exports = nconf;
\ No newline at end of file
+
+// @TODO: this file gets cached after the first require, we need to redesign how we load the config
+// @TODO: for example: require('ghost-ignition').config()
```

Comment:
To be clear, what I'm saying with 1. is that I think this fix looks like it's trying to patch a bad assumption, whereas I'd suggest we change or remove the assumption